sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSamplerProvider.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.extension.trace.jaeger.sampler;
+
+import io.opentelemetry.sdk.autoconfigure.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigurableSamplerProvider;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+import java.util.concurrent.TimeUnit;
+
+public class JaegerRemoteSamplerProvider implements ConfigurableSamplerProvider {
+
+  @Override
+  public Sampler createSampler(ConfigProperties config) {
+    String serviceName = config.getString("jaeger.service");
+    JaegerRemoteSamplerBuilder builder = JaegerRemoteSampler.builder().setServiceName(serviceName);
+
+    // Optional configuration
+    String endpoint = config.getString("jaeger.remote.sampler.endpoint");
+    if (endpoint != null) {
+      builder.setEndpoint(endpoint);
+    }
+    Double initialRate = config.getDouble("jaeger.remote.sampler.initial.rate");
+    if (initialRate != null) {
+      builder.setInitialSampler(Sampler.parentBased(Sampler.traceIdRatioBased(initialRate)));
+    }
+    Integer poolingInterval = config.getInt("jaeger.remote.sampler.pooling.interval");
+    if (poolingInterval != null) {
+      builder.setPollingInterval(poolingInterval, TimeUnit.MILLISECONDS);
+    }
+
+    return builder.build();
+  }
+
+  @Override
+  public String getName() {
+    return "jaeger_remote_sampler";
```

Comment:
How about moving this module to java-contrib? We're starting to use that repo more. We wouldn't need to be as strict about the spec there. Otherwise it does seem like we would want to add this Sampler to the spec.

Reply:
The goal is to include it in the official Java agent distribution. I would say modules from `java-contrib` do not make it that far. 

Note that Jaeger remote sampler has been here in the main java repository for a long time, but it's not included in the auto-instrumentation.

Reply:
Ah I didn't know that's your plan - in that case, it's even more important to go through spec. The actual sampler implementation doesn't need to be added, just add the variables

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#general-sdk-configuration

I think it could be similar to the propagators, `"jaeger": Link to Jaeger remote sampler doc` and a section with its endpoint-specific variables.

Since this artifact is already a stable one, we shouldn't add something like this name that we may need to change after the spec so can't merge this until the spec is updated

Reply:
> Ah I didn't know that's your plan - in that case, it's even more important to go through spec. The actual sampler implementation doesn't need to be added, just add the variables

yeah it is in the PR description :)

Reply:
Yeah missed it - but while it's nice we could reuse many variables, still we've defined this new env variable value here. We need it in the spec - since you've talked to @tedsuo and @yurishkuro already presumably the spec PR can be merged relatively quickly since there already seems to be consensus for it though we'll all want to come up with a name (I wouldn't repeat the word `sampler` in this value)

Reply:
didn't this end up being "jaeger_remote" in the spec PR that got merged?